app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -23,6 +24,7 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
     private lateinit var mcmickenMarker: Marker
     private lateinit var braunsteinMarker: Marker
     private lateinit var daapMarker: Marker
+    private var markers: MutableLiveData<ArrayList<Marker>> = MutableLiveData()
```

Comment:
Instead of creating a new property to hold a marker for every new location, you can create a livedata object with a list of markers, and any other class that needs the markers can simply observer this object.